src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -146,6 +154,7 @@ export function CurrencySearch({
 
   const [searchQuery, setSearchQuery] = useState<string>('')
   const debouncedQuery = useDebounce(searchQuery, 200)
+  const isQueryValidEVMAddress = isEVM(chainId) && !!isAddress(chainId, debouncedQuery)
```

Comment:
why only evm? solana racist?

Reply:
We will do later. Now we support EVM only